qubesadmin/tools/qvm_start_gui.py

```diff
@@ -239,27 +249,32 @@ def start_gui_for_stubdomain(self, vm, force=False):
 
     @asyncio.coroutine
     def start_gui(self, vm, force_stubdom=False, monitor_layout=None):
-        '''Start GUI daemon regardless of start event.
+        """Start GUI daemon regardless of start event.
 
         This function is a coroutine.
 
         :param vm: VM for which GUI daemon should be started
         :param force_stubdom: Force GUI daemon for stubdomain, even if the
             one for target AppVM is running.
-        '''
+        :param monitor_layout: monitor layout configuration
+        """
+        if vm.guivm != vm.app.get_local_name():
```

Comment:
This may be ok to leave (it's reachable directly via command line), but in this case, it should log some message why it's skipping the VM. Otherwise the call `qvm-start-gui abc` will be confusing why it does nothing.